test/extended/testdata/verifyservice-pipeline-template.yaml

```diff
@@ -0,0 +1,93 @@
+apiVersion: v1
+kind: Template
+labels:
+  template: nodejs-example
+metadata:
+  name: nodejs-example
+objects:
+- apiVersion: v1
+  kind: Service
+  metadata:
+    name: nodejs-example
+  spec:
+    ports:
+    - name: web
+      port: 8080
+      targetPort: 8080
+    selector:
+      name: nodejs-example
+- apiVersion: v1
+  kind: Service
+  metadata:
+    name: nodejs-example-headless
+  spec:
+    clusterIP: None
+    ports:
+    - port: 8080
+      targetPort: 8080
+    selector:
+      name: nodejs-example
+- apiVersion: v1
+  kind: DeploymentConfig
+  metadata:
+    name: nodejs-example
+  spec:
+    replicas: 2
+    selector:
+      name: nodejs-example
+    strategy:
+      type: Rolling
+    template:
+      metadata:
+        labels:
+          name: nodejs-example
+        name: nodejs-example
+      spec:
+        containers:
+        - env: []
+          image: 'nodejs:latest'
```

Comment:
After removing the imagechage trigger, have added the image and the tag here.

Reply:
The tests were failing with error regarding not being able to find the nodejs image. After adding the imagechange trigger seems to have made it better.

@gabemontero Do you have any suggestion regarding any other change to the image attribute ?

Reply:
OK my recollections and what was needed/not needed were just faulty here @waveywaves  .... let's leave the ICT as you have it now / originally had it

apologies for the churn

the other reductions were useful

and perhaps some context ... there have been efforts in the past to minimize the size of the bindata as much as possible ... so when we introduce new yaml, we attempt to be diligent